cloud/microsoft/office365/onedrive/mode/siteusage.pm

```diff
@@ -262,12 +275,21 @@ sub manage_selection {
     my ($self, %options) = @_;
     
     $self->{active} = { active => 0, total => 0, report_date => '' };
-    $self->{global} = { storage_used_active => 0, storage_used_inactive => 0, file_count => 0, active_file_count => 0 };
+    $self->{global} = { storage_used_active => 0, storage_used_inactive => 0,
+                        file_count_active => 0, file_count_inactive => 0, active_file_count => 0 };
     $self->{sites} = {};
 
-    my $results = $options{custom}->office_get_onedrive_usage();
+    my $results = $options{custom}->office_get_onedrive_usage(param => "period='D7'");
+    my $results_daily = [];
+    if (scalar(@{$results})) {
+       $self->{active}->{report_date} = @{$results}[0]->{'Report Refresh Date'};
+       $results_daily = $options{custom}->office_get_onedrive_usage(param => "date=" . $self->{active}->{report_date});
+    }
+
+    foreach my $site (@{$results}, @{$results_daily}) {
+        # As it's used as the instance label, let's keep the URL as short as possible, removing its domain name...
```

Comment:
Same same

Reply:
As [above](https://github.com/centreon/centreon-plugins/pull/2042#discussion_r464734744), instances cleaning remove for all other modes in https://github.com/centreon/centreon-plugins/pull/2042/commits/181d60f5162147fc4b7a466057fdddcc20476906.